pkg/actors/actors.go

```diff
@@ -972,7 +972,21 @@ func (a *actorsRuntime) RegisterInternalActor(ctx context.Context, actorType str
 	return nil
 }
 
-func (a *actorsRuntime) GetActiveActorsCount(ctx context.Context) []*runtimev1pb.ActiveActorsCount {
+func (a *actorsRuntime) GetRuntimeStatus(ctx context.Context) *runtimev1pb.ActorRuntime {
+	// Do not populate RuntimeStatus, which will be populated by the runtime
+	res := &runtimev1pb.ActorRuntime{
+		ActiveActors: a.getActiveActorsCount(ctx),
+	}
+
+	if a.placement != nil {
+		res.HostReady = a.placement.ActorHostReady() && a.haveCompatibleStorage()
```

Comment:
either the name `a.placement.ActorHostReady()` is misleading or a host is considered ready only when there is compatible storage. 
In that case, a separate function should be there in this file calling both `a.placement.ActorHostReady()` and the `a.haveCompatibleStorage()` methods. 
Additionally, `a.placement.ActorHostReady()` should be renamed to something different.

Reply:
This is a good point. I think I overloaded the expression "actor host ready" indeed.

- For placement, the host is ready when the connection with placement is up and the app is healthy. In this case, "actor host ready" was probably not the correct name
- For the actor runtime, "actor host ready" was meant to indicate that the runtime can host actors.

I will rename the method in placement.

Reply:
Done